app/hub/Views/HomeView/HomeView.scss

```diff
@@ -16,6 +16,13 @@
 	padding-top: 45px;
 	padding-bottom: 25px;
 	color: $tundora;
+	.button {
+		&:not(.hollow):hover {
+			&:hover {
+				background-color: #0078CA;
+			}
+		}
+	}
```

Comment:
I couldn't understand quite what you intended here so I tried to simplify and I ?seem? to be getting the same behavior with 

```
  .button {
	&:hover {
	  background-color: #0078CA;
	}
  }
```

If the CSS does need to be the way you have it, can you explain why please? - or if not, could you simplify it as above?

Reply:
As mentioned in our chat, we've changed the font and border color of the hollow buttons to match the new hover color, and also removed the double hover

Reply:
Great, thanks again for clarifying